packages/synapse-react-client/src/lib/containers/TextField.tsx

```diff
@@ -35,28 +35,26 @@ type TextFieldProps = Pick<
  */
 export default function TextField(props: TextFieldProps) {
```

Comment:
Apparently Alina and I had separately tried to make an input field using MUI that matches designs.

Alina's solution used `styled` and was closer to the design, but wasn't quite using the right MUI components (e.g. the CSS and existing usage required nesting a `TextField` (which contains a `FormControl`) in the `StyledFormControl`.

I created this custom `TextField` component, which is closer to a re-creation of the MUI TextField but tailored for our design requirements. I don't think the actual implementation matched designs as closely, but it's an "all-in-one" component that itself handles the FormControl wrapper and the label.

Here I combined the two--use the styled FormControl and fix some of the components in this custom component.

I think ideally, our theme would be configured so that we could just use the MUI TextField component instead of a custom component or styled component, but we'll get there eventually.